scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -43,9 +43,12 @@ sudo yum install -y kernel-devel-$(uname -r) \
 sudo yum install -y cuda-drivers \
     cuda
 
-# The Fabric Manager service needs be started and enabled on EC2 P4d instances
+# The Fabric Manager service needs to be started and enabled on EC2 P4d instances
 # in order to configure NVLinks and NVSwitches
 sudo systemctl enable nvidia-fabricmanager
+# NVIDIA Persistence Daemon needs to be started and enabled on P5 instances
+# to make processing faster.
```

Comment:
nit: ~~to make processing faster~~ to maintain persistent software state in the NVIDIA driver.

(i found this in https://manpages.ubuntu.com/manpages/xenial/en/man1/alt-nvidia-361-persistenced.1.html)

Reply:
ack, I will modify this. Thank you.